### ISSUE TITLE
Some news about BT

### ISSUE THREAD

**Issue by remrem** [BlogoText/blogotext#373]
I have not managed BT seriously for a few months (personal life), until I have completely taken things in hand (question of days), a small point on the current situation can not hurt.

By order of priority :
- BT 3.7.7, well advanced (see the branch "fix"), I must resume my notes, and switch to the master branch. There are still some issues that have not been resolved.
- BT 3.8, a lot of things planned (maybe too much, see the next point), some are well advanced, I'll see after 3.7.7 to push on the dev branch what I have locally and put in place a repository to test the new update system.
- BT 3.8 & after, the development cycle is (too) long, I will review the roadmap to spread the issues on several versions and release updates a little more frequently. BT 3.8 will bring new features for addons and updates, it would not hurt that it happens quickly ;)

Questions, suggestions or ideas?
Thanks!

**Comment by B4rb3rouss** [comment]
Question : do you need any help somewhere ?
Good to read you ;)

**Comment by remrem** [comment]
yep, I'm not against a quick check of the changes before the release.
If you have the time, I will send you an email with a test protocol as soon as I finish making the point (today, if no hiccup...)

**Comment by B4rb3rouss** [comment]
No problem, I'll look into it with pleasure.

**Comment by B4rb3rouss** [comment]
ping ?
😉

**Comment by B4rb3rouss** [comment]
It's been a month now. I have a few time next days if you need help, writing on blogotext.org or anything...

**Comment by B4rb3rouss** [comment]
Hi ?
It seems difficult for you these days ?
Is it possible to have a release (without all the exciting improvements) of the current progress ?
